packages/utils/codegen-templates-scripts/webpack.config.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
     'graphql-codegen-core',
     'graphql-tag',
     'lodash',
-    'graphql-codegen-plugin-helpers'
+    'graphql-codegen-plugin-helpers',
+    'moment'
```

Comment:
As I said, now it's treated as an external module, not included in a bundle generated by webpack.